cmd/dfdaemon/app/init.go

```diff
@@ -58,14 +89,18 @@ func initDfdaemon(cfg config.Properties) error {
 			"ensure local repo %s exists", cfg.DFRepo,
 		)
 	}
+	cfg.SuperNodes = adjustSupernodeList(cfg.SuperNodes)
+	if stringutils.IsEmptyStr(cfg.LocalIP) {
+		cfg.LocalIP = checkConnectSupernode(cfg.SuperNodes)
+	}
 
 	go cleanLocalRepo(cfg.DFRepo)
 
-	dfgetVersion, err := exec.Command(cfg.DFPath, "version").CombinedOutput()
-	if err != nil {
-		return errors.Wrap(err, "get dfget version")
-	}
-	logrus.Infof("use %s from %s", bytes.TrimSpace(dfgetVersion), cfg.DFPath)
+	// dfgetVersion, err := exec.Command(cfg.DFPath, "version").CombinedOutput()
```

Comment:
just remove them if you think the code is useless

Reply:
done.